basics/installation/localhost.md

```diff
@@ -28,13 +28,21 @@ Here is a selection of free AMP installers:
 
 To install LAMP on your computer follow these steps (tested on Debian Buster).
 - Update your system
-  - `apt update`
+```bash
+apt update
+```
```

Comment:
Missing space

Reply:
bash block has left padding for inside command so I don't think we need space there. MySQL commands in bash block a few lines below use the same format. Here is the original PR I made for 8.x docs https://github.com/leemyongpakvn/docs/blob/8.x/basics/installation/localhost.md

Reply:
There are spaces in the [PR for 8.0](https://github.com/PrestaShop/docs/pull/1409/files#diff-d9a6ca3d6766249de84eeae59807d0a72e6c224e2fdd8f2f45f781b10e174ecdR31-R33)

When i merge your PR i have this 

![Screenshot (87)](https://user-images.githubusercontent.com/5064590/179391473-f2a6a3a1-9b15-4047-91e6-4e975f3f4eaf.png)

Reply:
I see the missing space clearly now. Thank you.